src/System.Security.Permissions/src/System/SystemException.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace System
```

Comment:
If we add temporary stuff like this, won't we find ourselves in a bind later when we add it to the correct place? If I understand right, the bot will run overnight and this contract will then "see" the correctly placed type, causing a build break. (That's when we're in master -otherwise the bot doesn't update the versions for us - if my understanding is right)

This particular one should get into System.Runtime.dll with [https://github.com/dotnet/corefx/issues/9854](https://github.com/dotnet/corefx/issues/9854)

With the current infrastructure, then, we are forced to work up the stack and this is blocked on SystemException (and SecurityElement). Does that sound right? Ugh

---
In reply to: [70822457](https://github.com/dotnet/corefx/pull/10049#discussion_r70822457) [](ancestors = 70822457)